bugtool/cmd/root.go

```diff
@@ -246,6 +267,11 @@ func copySystemInfo(cmdDir string) {
 		cmd := fmt.Sprintf("kubectl -n %s logs --previous -p %s", k8sNamespace, pod)
 		k8sCommands = append(k8sCommands, cmd)
 	}
+	k8sCommands = append(k8sCommands, "kubectl describe nodes")
+	k8sCommands = append(k8sCommands, "kubectl version")
+	k8sCommands = append(k8sCommands, "kubectl -n kube-system get pods")
+	k8sCommands = append(k8sCommands, "kubectl get pods,svc --all-namespaces")
+	k8sCommands = append(k8sCommands, "kubectl get version")
```

Comment:
append can take multiple entries, if you preferred that form:
```golang
	k8sCommands = append(k8sCommands,
		"kubectl describe nodes",
		"kubectl version",
		"kubectl -n kube-system get pods",
		"kubectl get pods,svc --all-namespaces",
		"kubectl get version")
```

Reply:
Fixed